integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -763,10 +763,9 @@ case class LoadTable(
 
 
       val columinar = sqlContext.getConf("carbon.is.columnar.storage", "true").toBoolean
-      val kettleHomePath = CarbonScalaUtil.getKettleHome(sqlContext)
 
       // TODO It will be removed after kettle is removed.
-      val useKettle = options.get("use_kettle") match {
+      val useKettle = options.get("useKettle") match {
```

Comment:
DataFrameWriter accept parameter `useKettle` eg.  writer.option("useKettle", "false").option("tempCSV","false").  Maybe we should keep consistent。